ocp_resources/node_network_configuration_policy.py

```diff
@@ -370,32 +370,26 @@ def _process_failed_status(self, failed_condition_reason):
 
         raise NNCPConfigurationFailed(f"Reason: {failed_condition_reason}\n{last_err_msg}")
 
+    @retry(wait_timeout=120, sleep=5)
```

Comment:
`120` is problematic, you drop the ability to get the `wait_timeout`  (`self.success_timeout`) from the user.
I don't think `@retry` fit here